state/indexer/block.go

```diff
@@ -24,4 +24,10 @@ type BlockIndexer interface {
 	Search(ctx context.Context, q *query.Query) ([]int64, error)
 
 	SetLogger(l log.Logger)
+
+	Prune(retainHeight int64) (int64, int64, error)
+
+	SetBlockIndexerRetainHeight(retainHeight int64) error
+
+	GetBlockIndexerRetainHeight() (int64, error)
```

Comment:
Do we need such long function names? Could we call these `SetRetainHeight` and `GetRetainHeight` instead? They are already associated with the `BlockIndexer` interface, so unless we're composing this interface with another one that has similarly named functions, these names seem too verbose.

Reply:
Done